### DISCUSSION TITLE
Available map projection methods in Carla

### DISCUSSION THREAD

**Discussion by bernvier** [carla-simulator/carla#6202] [Q&A]
Hello,
each openDrive map that can be used in Carla needs the geoReference tag in which the map projection is defined. Generally, the openDrive format accepts a lot of different projection types. 
In the default Carla maps I checked the Transverse Mercator projection was always used but I would like to use a different projection method for the openDrive map. For example the following tag that stands for a UTM projection:

+proj=utm +zone=30 +ellps=WGS84

If I change the mentioned tag in the .xodr file and try  to generate the map in the standalone mode with client.generate_opendrive_world() I get the following error message:

WARNING: cannot parse georeference: '+proj=utm +zone=30 +ellps=WGS84'. Using default values.

I tried a few different projections and it seems that Carla does not accept every projection which is available for the openDrive format. Based on what I tried, only the Transverse Mercator projection seems to work properly. Therefore, I would like to know which projection methods other than the Transverse Mercator projection does Carla accept ? If only the Transverse Mecator projection is accepted would it be possible to extend Carla to support all the projection methods ?
What I find strange is that no warning from Carla is generated if the following projection is used:

+proj=utm +lat_0=0 +lon_0=0

But those tags do not make sense for the UTM projection type so I am wondering if Carla switches then in the background back to its default Transverse Mercator projection and uses the mentioned +lat_0 and +lon_0 tags.

Thank you for your help.

**Comment by CHILLed144** [comment]
i have this same problem. trying to use the UTM projection gives me a WARNING and its ignored in parsing.
using carla-dev4.26.
ubuntu 20.04
